subiquity/curtin.py

```diff
@@ -71,6 +71,11 @@
 # Reworked for flake8, but it does make it harder to read.
 POST_INSTALL_LIST = [
     ("late_commands:"),
+    ("    05_clear_subiquity: curtin in-target -- "
```

Comment:
This is already appropriately layered so that the files *should not* be there normally, obviously a work in progress. However, we should still be careful and make sure the files are removed if they do happen to show up, as otherwise they would break the installed system.